dvc/command/install.py

```diff
@@ -13,7 +13,7 @@ def run(self):
         try:
             self.repo.install()
         except Exception:
-            logger.exception("failed to install dvc hooks")
+            logger.exception("Failed to install DVC Git hooks")
```

Comment:
most likely exceptions don't start with an upper case for a reason - this message just a small part of the whole thing when it's printed

Reply:
It comes after `ERROR:` (at least per https://github.com/iterative/dvc/pull/3156#discussion_r368929870). From what I understand, a full sentence is capitalized after `:`

Of course this is logger output so grammar isn't very important. As you guys prefer, I can revert all these, lmk pls.

Reply:
I guess [it's optional](https://www.grammarly.com/blog/capitalization-after-colons/) anyway. Reverted all these in 69c1aa5b.

Reply:
I would avoid touching this one for now, too many questions ...